draft-ietf-quic-applicability.md

```diff
@@ -487,6 +487,30 @@ Padding can also be used by an application to reduce leakage of
 information about the data that is sent. A QUIC implementation can expose an
 interface that allows an application layer to specify how to apply padding.
 
+# Error Handling
+
+QUIC recommends that endpoints that detect errors should signal the occurance to
+the peer. Errors can occur at the transport level and the application level.
+Transport errors, such as a protocol violation, affect the entire connection.
+Applications that use QUIC can define their own error detection and signalling
+(see, for example, {{Section 8 of QUIC-HTTP}}). Application errors can affect an
+entire connection or a single stream.
+
+QUIC defines an error code space that is used for error handling. QUIC
+encourages endpoints to use the most-specific code, although any applicable code
+is permitted including generic ones. Applications using QUIC can define an error
```

Comment:
```suggestion
is permitted, including generic ones.

Applications using QUIC define an error
```
It's not a "can," since the spaces are distinct.  The application defines any semantics that might exist.

Reply:
I think we actually need to go one step further. My original intent was to say "applications can (and probably should define some codes". But with the comments and suggestions in this PR, I think the clearer statement is more like

```
Applications using QUIC are responsible for defining the error codes used to signal connection-level or stream-level errors (see, for example, {{Section 8.1 of QUIC-HTTP}}). Error code values are independent from QUIC or other applications. Values can be shared between the error types.
```

Reply:
I'd be okay with that text.